GVFS/GVFS/CommandLine/ConfigVerb.cs

```diff
@@ -10,18 +11,34 @@ public class ConfigVerb : GVFSVerb.ForNoEnlistment
         private const string ConfigVerbName = "config";
         private LocalGVFSConfig localConfig;
 
+        [Option(
+            "list",
+            SetName = "needNoKeys",
+            Default = false,
+            Required = false,
+            HelpText = "Show all settings")]
+        public bool List { get; set; }
+
+        [Option(
+            "delete",
+            SetName = "needsKey",
+            Default = null,

```

Comment:
we don't need to specify Default.

This is how it looks - 
<img width="752" alt="screen shot 2018-10-15 at 12 19 57 pm" src="https://user-images.githubusercontent.com/378580/46979757-7325c500-d0a0-11e8-9e1d-eec801c59d14.png">

Reply:
Re: that latest screenshot - it looks like config is out of order because the rest of the list is alphabetized. You probably need to change the order in Program.cs.

Reply:
right. re-ordered it.